src/plugins/chapters/index.js

```diff
@@ -172,7 +176,8 @@ const ChaptersPlugin = (function () {
   ChaptersPlugin.prototype.setupProgressBarMarkers = function setupProgressBarMarkers() {
     const total = this.player.duration();
     const { seekBar } = this.player.controlBar.progressControl;
-    this.chaptersTrack.cues_.forEach(marker => {
+
+    Array.from(this.chaptersTrack.cues).forEach(marker => { // Safari needs Array.from()
```

Comment:
same comment here you are now accessing a new property `.cues` instead of `.cues_`, please check that the Array.from is indeed needed. if not then removing the Array.from and the comment makes the code easier to understand and makes it look less fragile.